packages/account/src/Components/Routes/binary-routes.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+// TODO use deriv/shared once routes-config are merged.
+import getRoutesConfig from 'Constants/routes-config';
```

Comment:
We can use this from Shared now.

Reply:
Not this one, I've made a mistake commenting this because I thought route configs are also moved to shared. I'll remove the comment.